File: codex_poting/java/src/main/java/com/example/demo/service/PurchaseOperationService.java
package com.example.demo.service;

import com.example.demo.api.request.PurchaseInvoiceCreateRequest;
import com.example.demo.api.response.PurchaseInvoiceResponse;
import lombok.NonNull;

public interface PurchaseOperationService {
    PurchaseInvoiceResponse create(@NonNull PurchaseInvoiceCreateRequest request) ;
}

File: codex_poting/kotlin/src/main/kotlin/com/example/demo/service/PurchaseOperationService.kt
package com.example.demo.service

import com.example.demo.api.request.PurchaseInvoiceCreateRequest
import com.example.demo.api.response.PurchaseInvoiceResponse

interface PurchaseOperationService {
    fun create(request: PurchaseInvoiceCreateRequest): PurchaseInvoiceResponse?
}
